### ISSUE TITLE
Cannot add items to MATE panel

### ISSUE THREAD

**Issue by pccobbler** [geckolinux/geckolinux-project#143]
First, let me say that using Brisk Menu was the correct thing to do, as it's the best MATE menu.

There's a bug in adding items to MATE panel. I just built a Gecko MATE 15.2 system. I did "zypper update" so everything should be current. I tried the usual:
- right-click on the panel and select "Add to Panel ..."
- Select "Application Launcher"
- Select "Add"

However, on the subsequent popup, all of the items are unselectable, i.e. grayed-out. I've never seen this behavior before.

**Comment by geckolinux** [comment]
Hi, is this with the ROLLING or the STATIC edition? I confirm this bug on ROLLING.

I wonder if it's because of the Brisk menu being used?

At any rate, an easy workaround is to drag'n'drop the launcher from the Brisk menu to the panel.

**Comment by pccobbler** [comment]
I'm using STATIC. The Brisk Menu could be the problem, as I had to manually update it in Linux Mint 19 to get it to even install (see the URL below for that thread). That said, I've used MATE -- it's my favorite desktop now that Cinnamon has abandoned metacity-theme-1.xml format -- on other distributions but I've not seen this particular bug before.

https://forums.linuxmint.com/viewtopic.php?f=206&t=313059&p=1769698

Let me try that workaround.

UPDATE: Yes, that works fine. And it's easier. I did not know that could be done. It's Cinnamon-like, instead of the method I was trying which is LXDE/LXQt-like. Thanks.
